sql-db/sql-reactive/src/test/java/io/quarkus/ts/openshift/reactive/MsSQLDatabaseIT.java

```diff
@@ -0,0 +1,97 @@
+import io.vertx.mssqlclient.MSSQLConnection;
+
+@QuarkusScenario
+public class MsSQLDatabaseIT extends AbstractReactiveDatabaseIT {
+
+    private static final String MSSQL_USER = "sa";
+    private static final String MSSQL_PASSWORD = "QuArKuS_tEsT";
+    private static final String MSSQL_DATABASE = "msdb";
+    private static final int MSSQL_PORT = 1433;
+
+    @Container(image = "${mssql.image}", port = MSSQL_PORT, expectedLog = "Service Broker manager has started")
+    static DefaultService database = new DefaultService()
+            .withProperty("ACCEPT_EULA", "Y")
+            .withProperty("SA_PASSWORD", MSSQL_PASSWORD);
+
+    @QuarkusApplication
+    static RestService app = new RestService().withProperties("mssql.properties")
+            .withProperty("quarkus.datasource.username", MSSQL_USER)
+            .withProperty("quarkus.datasource.password", MSSQL_PASSWORD)
+            .withProperty("quarkus.datasource.reactive.url",
+                    () -> "vertx-reactive:" + database.getHost().replace("http", "sqlserver") + ":" + database.getPort() + "/"
+                            + MSSQL_DATABASE);
+
+    @Override
+    protected RestService getApp() {
+        return app;
+    }
+
+    @Test
+    void jdbc() throws SQLException {
+        Properties connectionProps = new Properties();
+        connectionProps.put("user", MSSQL_USER);
+        connectionProps.put("password", MSSQL_PASSWORD);
+        connectionProps.put("databaseName", MSSQL_DATABASE);
+        Connection connection = DriverManager
+                .getConnection(database.getHost().replace("http", "jdbc:sqlserver") + ":" + database.getPort(),
+                        connectionProps);
+        try (Statement statement = connection.createStatement()) {
+            ResultSet set = statement.executeQuery("SELECT title from books where id=1");
+            set.next();
+            String title = set.getString("title");
+            Assertions.assertEquals("Slovník", title);
+        }
+    }
+
+    @Test
+    void vertx() {
+        MSSQLConnectOptions options = new MSSQLConnectOptions(new JsonObject()
+                .put("jdbcUrl",
+                        database.getHost().replace("http", "jdbc:sqlserver") + ":" + database.getPort() + "/" + MSSQL_DATABASE)
+                .put("principal", MSSQL_USER)
+                .put("credential", MSSQL_PASSWORD));
+        MSSQLConnection.connect(Vertx.vertx(), options)
+                .flatMap(connection -> connection.query("SELECT title from books where id=1").execute())
+                .map(set -> set.iterator().next())
+                .map(row -> row.getString("title"))
+                .onSuccess(title -> {
+                    Assertions.assertEquals("Slovník", "title");
+                })
+                .onFailure(Assertions::fail);
+    }
+
+    @Test
+    @Override
+    @Disabled
+    public void testUniEndpoint() {
+        // MSSQL hibernate has some problems with extended Latin
```

Comment:
This sounds odd to me. Can you describe a bit more the problems or add a link to the issue to get more information? Same as below.

Reply:
Yes, tests testUniEndpoint and testMultiEndpoint fail on retrieval of word "Slovník" from the database. The sixth letter(from extended Latin set) is replaced with some unreadable symbol. There is no issue AFAIK, since I found this behaviour while making this test.

Reply:
Added issue link to the code.